Posit Infrastructure/FAQs.md

```diff
@@ -149,6 +151,19 @@ Doing so may result in Posit Workbench crashing or certain error messages such a
 
 This is a [known issue](https://github.com/rstudio/rstudio/issues/11914) in older versions of Posit Workbench which will likely be fixed when the Posit Workbench environment is next updated.
 
+### Troubleshooting packages
+
+#### How do I stop `{shiny}` apps timing out so quickly?
+
+If your `{shiny}` apps are timing out after just a few seconds when run locally, you can add the workaround code below to the app's server.R script to prevent this:
+
```

Comment:
Should people know *why* this code works or is it fine to just tell them to use it without much question?

Reply:
Since it's intended to be a workaround and will only show in the console in the background when run, I'd say it's fine to keep it quite general. @mgannon99 might feel differently though!

Reply:
Probably best to keep it general at the moment.